### PR TITLE
FEAT: OmniSciDB - Add support for within, d_fully_within and point

### LINKED ISSUE #1924: FEAT: Add new geospatial operations into OmniSciDB/MapD backend
The follow operations is already available in OmniSciDB and should be added into ibis.omniscidb backend.

- [x] ST_INTERSECTS
- [x] ST_DWITHIN
- [x] ST_DISJOINT
- [ ] ST_DFULLYWITHIN
- [ ] ST_POINT
- [ ] ST_WITHIN

### PR DESCRIPTION
**Pull request by xmnlab**
Resolves #1924

### CODE REVIEW

**Comment by xmnlab** [comment]
@jreback it is ready for review. thanks!

**Comment by jreback** [comment]
does this fully close #1924?

**Comment by xmnlab** [comment]
thanks for the review @jreback ! yes it fully resolves #1924 
release note added.

**Comment by xmnlab** [comment]
ci is green

File: ibis/expr/api.py [hunk 1]
```diff
@@ -2024,6 +2026,24 @@ def geo_overlaps(left, right):
     return op.to_expr()
 
 
+def geo_point(left, right):
```
Comment (jreback) [inline ibis/expr/api.py]: can you type

**Comment by jreback** [comment]
thanks @xmnlab

**Pull request merged**